### DISCUSSION TITLE
could you please share the collection setup shown in git-readme ?

### DISCUSSION THREAD

**Discussion by snickers2k** [meisnate12/Plex-Meta-Manager#599] [Q&A] [answered]
<img width="1633" alt="collections1-1" src="https://user-images.githubusercontent.com/5986024/149032537-1a5268ba-f35c-4d4e-ba35-c37a95e0f49f.png">

this looks pretty awesome. you @meisnate12  probably took the tmdb network/collections icons automatically as an overlay right?

can't find anything in your config shared in configs-repo. do you do this by a global template? could you please share this one? 

thanks

**Comment by meisnate12** [comment, marked as answer]
those collections are from yozora hes one of the discord moderators if you've got any questions about his configs which are in the repo under his name i guess i should include a link under the pic saying where they're from.

https://github.com/meisnate12/Plex-Meta-Manager-Configs/tree/master/yozora

**Comment by YozoraXCII** [comment]
My poster set has been uploaded to the PMM Config repo here:
https://github.com/meisnate12/Plex-Meta-Manager-Configs/tree/master/yozora

You will find all my posters there as well as a download link.

**Comment by snickers2k** [comment]
ah i see. thank you guys! :)

**Reply by YozoraXCII** [reply]
> ah i see. thank you guys! :)

Poster uploads are not allowed within the repo, so you can grab a live copy here:
https://mega.nz/folder/RcE1VIZQ#WTwcOGwETnWZEjafLsIFHA

**Reply by snickers2k** [reply]
thats awesome! thank you.
i really love the clean look per type. makes it so much easier to have an overview. 👍 

but thats a lot of photoshop hustle :) of course many of them obviously had to be made by yourself 

But because i have lots of German Networks aswell,
maybe it's worth a feature request - to being able to take network and company logos, as an overlay ?

like

```
Likasfilm Collection:
  tmdb_studio: 1
  url_poster: .../plain_colored_poster_background.png
  item_overlay: 
    tmdb_studio: 1
```

i really like automation :D
obviously to not have to start photoshop :D but even more if any icon has changed, it would automatically change in plex aswell.

**Reply by YozoraXCII** [reply]
> thats awesome! thank you. i really love the clean look per type. makes it so much easier to have an overview. 👍
> 
> but thats a lot of photoshop hustle :) of course many of them obviously had to be made by yourself
> 
> But because i have lots of German Networks aswell, maybe it's worth a feature request - to being able to take network and company logos, as an overlay ?
> 
> like
> 
> ```
> Likasfilm Collection:
>   tmdb_studio: 1
>   url_poster: .../plain_colored_poster_background.png
>   item_overlay: 
>     tmdb_studio: 1
> ```
> 
> i really like automation :D obviously to not have to start photoshop :D but even more if any icon has changed, it would automatically change in plex aswell.

There are ongoing discussions around this exact thing - I can't guarantee anything, but watch this space.
